tests/Database/SelectFields/InterfaceTests/InterfaceTest.php

```diff
@@ -47,6 +48,58 @@ public function testGeneratedSqlQuery(): void
         $this->assertSame($expectedResult, $result);
     }
 
+    public function testGeneratedRelationSqlQuery()
```

Comment:
Please always add the return type (`: void` in this case), like we do it everywhere, thank you!